build.js

```diff
@@ -54,7 +54,7 @@ const supportedUtilities = [
 	// Height
 	/^(h-\d+|h-px|h-full)/,
 	// Min/Max width/height
-	/^(min-w-|max-w-|min-h-0|min-h-full|max-h-full)/,
+	/^(min-w-|max-w-|min-h-0|min-h-full|max-h-full|max-h-\d+)/,
```

Comment:
Could you replace this with a more lax regex to match all strings, not just numbers?

```suggestion
	/^(min-w-|max-w-|min-h-0|min-h-full|max-h-)/,
```

You'd also need to exclude `max-h-screen` (it's not supported) by adding a new `if` condition here https://github.com/vadimdemedes/tailwind-rn/blob/master/build.js#L92.

Reply:
I went ahead and also modified `min-h-`, since those are basically the same, and excluded both `min-h-screen` and `max-h-screen`.